confidential-data-hub/hub/src/config.rs

```diff
@@ -95,6 +159,21 @@ impl CdhConfig {
     fn from_file(config_path: &str) -> Result<Self> {
         let c = Config::builder()
             .set_default("socket", DEFAULT_CDH_SOCKET_ADDR)?
+            .set_default("image.security_validate", "true")?
+            .set_default("image.auth", "true")?
```

Comment:
Should auth default to true? If it is set to true and `kbs:///default/credential/test` hasn't been set will it error, or just ignore the "empty" creds?

Reply:
Oh. In kata we mark it as true by default [here](https://github.com/kata-containers/kata-containers/blob/main/src/agent/src/image.rs#L64). And with [this fix](https://github.com/confidential-containers/guest-components/pull/620/files), we'd better mark it by default false as you suggested. Thanks.

Reply:
Yeah, I think in `CCv0` we just said if we used the KBS at all we made the credentials check, so it was true a lot of the time and we'd handle missing creds gracefully, but now it fails (which is good for error checking), then we want to deliberately enable it when expected now I guess.